IDEAS.md

```diff
@@ -1,120 +1,173 @@
-# IDEAS for the future of Gupaxx
+# **IDEAS for the future of Gupaxx**
 
 
 **Theses are only ideas, everything here is still to be decided and only thoughts for now.**
-Some ideas could be done in a matter of hours, some could take months.
+- Note: Some ideas could be done in a matter of hours, some could take months.
 
-## More Decentralized
+## **Making Gupaxx more Decentralized**
 ### Synchronize source code repository on p2p network
-Github is proprietary. If Gupaxx aims to be free, it should not be only available on this platform and we should explore options to get github free.  
-We can use [Radicle](https://radicle.xyz/) to get Gupaxx on a p2p collaboration stack.  
-The code, issues and PR could be synchronized with Github.  
+- Github is proprietary. If Gupaxx aims to be free software, it should not be only available on this proprietary platform and we should explore options to be github free.  
+- We can use [Radicle](https://radicle.xyz/) to get Gupaxx on a p2p collaboration stack.  
+- We can use [Codeberg](https://codeberg.org/) as an open source alternative for Github.
```

Comment:
Please keep commits for grammar/formatting separated with new ideas.

Reply:
will create seperate pr for grammar and formatting when this pr is merged/accepted.